src/backend/joanie/payment/backends/payplug/__init__.py

```diff
@@ -244,3 +253,8 @@ def abort_payment(self, payment_id):
             payplug.Payment.abort(payment_id)
         except (Forbidden, NotFound) as error:
             raise exceptions.AbortPaymentFailed(str(error)) from error
+
+    def cancel_or_refund(self, amount, transaction_reference):
```

Comment:
If it should be not implemented, then you should not override and keep the default behavior